### ISSUE TITLE
TRAP_CAUSE_UNKNOWN bugcheck 100% occurs on latest build in win7 sp1 32-bit

### ISSUE THREAD

**Issue by deserwild** [tandasat/HyperPlatform#38]
### Steps to reproduce the problem

1. Compile the project.
2. Copy HyperPlatform.sys to target machine.
3. Load HyperPlatform.sys.
4. Bugcheck occurs immediately.

### Specifications
- Commit 643b81c

- OS version 7601 

- Architecture: Only x86

- Hardware Physical & VMware 12

- Details:

It seems that exceptions was generated by "movaps" instruction which is newly added to x86.asm. Dmp&pdb&log was sent to your gmail.

Thanks!

**Comment by asamy** [comment]
Not sure but maybe `movdqa` could be a potential fix?

**Comment by tandasat** [comment]
Thanks for reporting the critical issue. I quickly confirmed it (sorry I thought I tested on x86 but I probably missed). 

One of issues was that we had to clear some flags in CR0 before touching floating point registers. Even after this fix, I realized that Internet Exlorer never ran successfully on Win7 x86 when movaps to xmmN was active. As I was not sure what the problem was, I made a quick workaround for now. 

I will investigate the cause shortly.
